ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.12-postsubmits.yaml

```diff
@@ -54,7 +54,7 @@ postsubmits:
     always_run: true
     branches:
     - ^release-4\.12$
-    cluster: build03
+    cluster: build01
```

Comment:
Is this an automatic change?

Reply:
Yes, this was an automatic change after running `make jobs `

Reply:
I think this is an automated change because of a stale `make jobs` on the previous commit.  `build03` is added in [commit](https://github.com/openshift/release/pull/36314/commits/0f1b2271f28e256f95bf5888e6b36aaf10639463) and then changed to `build01` in the following commit.

Reply:
Nit: Then this change should have been in the `fast-forward community-4.12 branch` commit